Un-deprecate scrollable alert dialog by default (#5240)
flutter/website: src/docs/release/breaking-changes/scrollable-alert-dialog.md

---
title: Scrollable AlertDialog (No longer deprecated)
description: AlertDialog should scroll automatically when it overflows.
---

## Summary

> Note: `AlertDialog.scrollable` is no longer deprecated since there is
> no backwards compatible way to have AlertDialog be scrollable by default.
> Instead, the parameter will remain and users will be able to set `scrollable`
> to true if they want a scrollable `AlertDialog`.
> 

Reviewer: ```suggestion
{{{site.alert.note}}
  `AlertDialog.scrollable` is no longer deprecated because there is
  no backwards-compatible way to make `AlertDialog` scrollable by default.
  Instead, the parameter will remain and you can set `scrollable`
  to true if you want a scrollable `AlertDialog`.
{{site.alert.end}}
```
